src/constants.ts

```diff
@@ -5,7 +5,9 @@ export type BigintIsh = JSBI | bigint | string
 
 export enum ChainId {
   MAINNET = 56,
-  BSCTESTNET = 97
+  BSCTESTNET = 97,
+  ETHMAINNET = 1,
+  ETHTESTNET = 1,
```

Comment:
Kuys same po talaga ETH test net chain id sa mainnet? Pwede man natin i 5 sa testnet for goerli?

Reply:
aw haneps hahaha